modules/integration/src/test/scala/scala/cli/integration/MavenTestHelper.scala

```diff
@@ -0,0 +1,6 @@
+package scala.cli.integration
+
+trait MavenTestHelper {
+
+  protected def mavenCommand(args: String*): os.proc = os.proc("ls") // os.proc(sbt, args)
```

Comment:
We'll need to run the exported project with maven here (I know, stating the obvious, I know it's WIP, just noting)